api/v1/badge/repository.go

```diff
@@ -112,21 +112,19 @@ func GetDefaultLevel(bdg *v1.Badge) (*v1.BadgeLevel, error) {
 	if err != nil {
 		return nil, err
 	}
-	var found *v1.BadgeLevel
+	var found v1.BadgeLevel
 	for _, lvl := range levels {
-		if lvl.IsDefault {
-			if found != nil {
-				// We have two default levels -> error
+		if lvl.IsDefault == true {
+			if found.ID != "" {
 				return nil, errors.BadRequestf("there are more than one levels with `isdefault=true`")
```

Comment:
I know it's legacy, but this doesn't sound like a *Bad Request*, since the user is probably not able to fix it himself. Opportunity to fix this? 🙂